### PR TITLE
add a form to submit comments on the home page

### PR DESCRIPTION
**Pull request by GovernessS**
created a comments submission form on index.html, where accessing /data will show an ArrayList of all submitted comments in one session

(The location of the form submission is not ideal, and I'll fix in a later iteration - I just wanted to get the comments section working)

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,35 +27,38 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> funFacts = new ArrayList<String>();
+  private ArrayList<String> comments = new ArrayList<String>();
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Change this to List<String> comment = ..

List is the interface & ArrayList is an implementation of List. If you use List, at a later time you can change the implementation to use SortedList or something and not impact any of the code.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -27,35 +27,38 @@
...
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      funFacts.add("I used to read the dictionary for fun!");
-      funFacts.add("I considered going to the University of Madison (Better Dead Than Red)");
-      funFacts.add("I spent over $100 on a planner in the 10th grade!");
+    response.setContentType("text/html;");
+    response.getWriter().println(comments);
+  }
+
+
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: formatting nit: Use 1 space between methods. You can use the linter I shared and that should do all the formatting for you.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -143,3 +143,10 @@ ol.roman-list {
   width: 200px;
 }
 
+#comments {
+  margin-left: 40%;
+  margin-right: 30%;
+  margin-top: 300px;
+  position: absolute;
+}
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css:151]: Formatting needs some work. The images are overlapping the comments section. You can fix that in a follow up PR.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (GovernessS) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (GovernessS) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: Done! And I'll be using the linter moving forward

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (GovernessS) [reply portfolio/src/main/webapp/style.css:151]: Yeah, it's something I'll fix once I spend more time on styling! Just wanted to get the functionality working first.

**Pull request merged**